openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -660,6 +660,17 @@ def build_anim(project_name, asset_name):
         bpy.data.collections.remove(gdeform_collection)
     create_gdeformer_collection(bpy.context.scene.collection)
 
+    # Get world from setdress
+    setdress_world = None
+    for container in bpy.context.scene.openpype_containers:
+        if container.get("avalon", {}).get("family") == "setdress":
+            for d_ref in container.datablock_refs:
+                if isinstance(d_ref.datablock, bpy.types.World):
+                    setdress_world = d_ref.datablock
+
+    # Assign setdress or last loaded world
+    bpy.context.scene.world = setdress_world or bpy.data.worlds[-1]
```

Comment:
Maybe we should but `None` if not found, this way it'll be obvious something's wrong. What do you think?